lms/tasks/email_digests.py

```diff
@@ -93,7 +92,9 @@ def send_instructor_email_digest_tasks():
                     LTIRole.type == "instructor",
                     not_(
                         UserPreferences.preferences[
-                            f"{EmailPreferencesService.KEY_PREFIX}{weekday}"
+                            EmailPreferences.user_preferences_key_for_email_digest_date(
```

Comment:
This is a bit more verbose but I think a bit nicer to have this little function to explicitly get the right key for the setting in the DB.